src/device-registry/models/Event.js

```diff
@@ -1551,7 +1551,7 @@ eventSchema.statics = {
             foreignField: "_id",
             as: "device_details",
           })
-          .match({ "device_details.visibility": visibilityFilter })
+          // .match({ "device_details.visibility": visibilityFilter })
```

Comment:
Hi @BenjaminSsempala , please take note of this commented out code in case such a request comes back in the near future from the hardware team. At the moment, they seem to be undecided hence the reason why we are reversing a change they initially asked us to make. :) cc: @Codebmk